creusot-contracts/src/std/array.rs

```diff
@@ -83,4 +83,10 @@ extern_spec! {
         #[ensures(self@ == result@)]
         fn into_iter(self) -> std::array::IntoIter<T, N>;
     }
+
+    impl<T: Clone, const N: usize> Clone for [T; N] {
+        #[ensures(forall<i> 0 <= i && i < self@.len() ==>
+            T::clone.postcondition((&self@[i],), result@[i]))]
```

Comment:
We could specify `Copy` as implying a trivial postcondition (since it's meant to capture "just a memcpy")

Reply:
Ideally, we would want to add this as a law of `Copy`, but we don't have an easy way to do this....